### PR TITLE
Add custom preference center

### PR DESCRIPTION
**Pull request by khmMouna**
### What do these changes do?
Add custom preference center screen

### CODE REVIEW

File: example/lib/screens/preference_center.dart [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    ScopedSubscriptionListEditor editor = Airship.editContactSubscriptionLists();
+    if (isSubscribed) {
+      editor.subscribe(subscriptionId, scopes);
+      activeContactSubscriptions.putIfAbsent(subscriptionId, () => scopes);
+    } else {
+      editor.unsubscribe(subscriptionId, scopes);
+      activeContactSubscriptions.remove(subscriptionId);
+    }
+    editor.apply();
+    //setState(() {});
```
Comment (jyaganeh) [inline example/lib/screens/preference_center.dart:90]: should this be uncommented or removed?

File: example/lib/screens/preference_center.dart [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    return ListTile(
+        title: Text('${item.display.title}', style:TextStyle(fontWeight: FontWeight.bold)),
+        subtitle: Text('${item.display.subtitle}'),
+    );
+  }
+
+  Widget item(IndexPath indexPath) {
+    List<PreferenceCenterItem> items = preferenceCenterConfig.sections[indexPath.section].items;
+    PreferenceCenterItem item = items[indexPath.item];
+    switch (item.type) {
```
Comment (jyaganeh) [inline example/lib/screens/preference_center.dart:171]: CI is complaining about a missing return for this method if we don't match any of the cases. Might need to throw or return a null that we can ignore higher up?

**Review by jyaganeh — APPROVED** [review]
Nice! The sample seems easy to understand and tweak 👍

File: example/lib/screens/preference_center.dart (hunk 1 above)
Comment (khmMouna) [reply example/lib/screens/preference_center.dart]: Yes 👍  it should be uncommented. I did some other improvements if you want to take a look

**Review by jyaganeh — APPROVED** [review]
👍

**Pull request merged**